app/src/main/java/de/westnordost/streetcomplete/quests/police_type/AddPoliceTypeForm.kt

```diff
@@ -0,0 +1,33 @@
+package de.westnordost.streetcomplete.quests.police_type
+
+import android.os.Bundle
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.quests.AImageListQuestAnswerFragment
+import de.westnordost.streetcomplete.view.image_select.Item
+
+class AddPoliceTypeForm : AImageListQuestAnswerFragment<PoliceType, PoliceType>() {
+
+    override val items = listOf(
+        Item(PoliceType.CARABINIERI,   R.drawable.ic_police_type_carabinieri,           R.string.quest_policeType_type_carabinieri),
+        Item(PoliceType.POLIZIA_DI_STATO,       R.drawable.ic_police_type_polizia,             R.string.quest_policeType_type_polizia_di_stato),
+        Item(PoliceType.POLIZIA_MUNICIPALE,     R.drawable.ic_police_type_municipale,          R.string.quest_policeType_type_polizia_municipale),
+        Item(PoliceType.POLIZIA_LOCALE, R.drawable.ic_police_type_locale, R.string.quest_policeType_type_polizia_locale),
+        Item(PoliceType.GUARDIA_DI_FINANZA,      R.drawable.ic_police_type_finanza,           R.string.quest_policeType_type_guardia_di_finanza),
+        Item(PoliceType.POLIZIA_STRADALE,    R.drawable.ic_police_type_stradale,         R.string.quest_policeType_type_polizia_stradale),
+        Item(PoliceType.GUARDIA_COSTIERA,       R.drawable.ic_police_type_costiera,             R.string.quest_policeType_type_guardia_costiera),
+        Item(PoliceType.POLIZIA_FERROVIARIA,    R.drawable.ic_police_type_ferroviaria,         R.string.quest_policeType_type_polizia_ferroviaria)
```

Comment:
I think the columns here should either be aligned with spaces, or not at all (just a single space after each comma).

Reply:
I'll try to align them in columns